Dockerfile

```diff
@@ -0,0 +1,6 @@
+
+FROM openjdk:17-slim
+
+ADD ./target/chat-dancer.jar /chat-dancer.jar
+CMD ["java",  "-jar", "/chat-dancer.jar"]
```

Comment:
This will not work, your current build specification is not producing this file. See how this is done in the dancer:

https://github.com/dancier/dancer/blob/b021ea96aa2e23c839ab3975b1866831341509ef/pom.xml#L143